datafeeder/src/main/java/org/georchestra/datafeeder/autoconf/GeorchestraIntegrationAutoConfiguration.java

```diff
@@ -15,8 +16,8 @@
  */
 @Configuration
 @Profile("georchestra")
-@Import(GeorchestraDatadirConfiguration.class)
-public class GeorchestraIntegrationConfiguration {
+@Import({ GeorchestraDatadirConfiguration.class, GeorchestraSecurityProxyAuthenticationConfiguration.class })
```

Comment:
for my own knowledge: what is the purpose of the `@Import` spring annotation ? I can read on the net that it helps grouping configuration classes, but I'd expect spring to scan the classpath for @configuration annotated classes and do the job of resolving the dependency graph by itself, without the need of using these imports.

Reply:
your expectation is based on "old fashion" spring "scan all classpath" approach though, which is actually an anti-pattern, besides time-consuming. I'd rather know what exactly is gonna be configured by grouping configuration classes hierarchically, and enabling them using profiles where appropriate. In this app we even use our own auto-configuration classes to automatically set up the required environment to work as a georchestra app or not, based on the active profile.

Reply:
ok makes sense